src/main/java/org/example/homework_nr_4/HomeWork4.java

```diff
@@ -0,0 +1,83 @@
+package org.example.homework_nr_4;
+
+import static java.lang.System.arraycopy;
+
+public class HomeWork4 {
+
+    public static void main(String[] args) {
+// Задание 4
+// Инициализация одномерного массива
+        int[] array = new int[]{4, 0, -1, 5, 12, 23, 11};
+        for (int index = 0; index != array.length; index++) {
+//Вывод элементов массива на коноль
+            System.out.print(array[index]+"  ");
+        }
+
+        System.out.println();
+
+//Задание 5
+//        Инициализация одномерного массива типа int
+//                и двумерного массива типа int
+
+        int[] anotherarray = new int[]{9,8,7,6,5,4};
+
+            int [][] bidiamationarray = new int[][] {
+                    {3,7,14,8,-2},
+                    {9,35,1,88,1},
+                    {2,6,8,10,12}
+            };
+// Вычесление суммы и среднего значения элементов одномерного массива
+        float sum = 0;
+        float count=0;
+        float avarage =0;
+        for (int j = 0; j != anotherarray.length; j++) {
+              sum += anotherarray[j];
+        }
+        for (int j = 0; j != anotherarray.length; j++) {
+            count++;
```

Comment:
move that count into the loop above. no need to create one more loop when you can fit two instructions into one loop above

Reply:
was moved